tests/function_modifiers/test_adapters.py

```diff
@@ -457,6 +458,7 @@ def df(data: pd.DataFrame) -> pd.DataFrame:
     )
     # This is just reading the same file we wrote out, so it should be the same
     pd.testing.assert_frame_equal(result["df"], result_just_read["df"])
+"""
```

Comment:
Leave this test in. Since it's testing the `@save_to` and `@load_from` paths.

Reply:
Corrected in most recent PR and reinstated